styles/_pages.scss

```diff
@@ -712,10 +795,79 @@
       background-image: url(../img/cards/4col-light-green.svg);
     }
   }
+
+  #run-a-network-node {
+    padding-bottom: 5rem;
+
+    @include media-breakpoint-up(md) {
+      padding-top: 104px;
+      padding-bottom: 104px;
+    }
+
+
+    .text-cards {
+      grid-gap: 40px;
+
+      h6::before {
+        margin-top: 0;
+        height: unset;
+      }
+
+      a {
+        font-size: 1.25rem;
+        line-height: 26px;
+        color: #fff;
+        font-weight: bold;
+
+        &:hover { 
+          text-decoration: none;
+          background: none !important;
+        }
+      }
+
+      .btn-arrow::after {
+        display: inline-block;
+        content: url(../img/icons/arrow-right-purple.svg);
+        vertical-align: middle;
+        padding-left: 8px;
+        -webkit-transition: transform 0.3s ease-out;
+        -moz-transition: transform 0.3s ease-out;
+        -ms-transition: transform 0.3s ease-out;
+        -o-transition: transform 0.3s ease-out;
+        transition: transform 0.3s ease-out;
+      }
+    }
+  }
+
+  #xrpl-grants {
+    padding-bottom: 5rem;
+    @include media-breakpoint-up(md) {
+      padding-top: 104px;
+      padding-bottom: 104px;
+    }
+  }
+
+  #xrpl-events {
+    padding-bottom: 5rem;
+    @include media-breakpoint-up(md) {
+      padding-top: 104px;
+      padding-bottom: 104px;
+    }
+  }
+
+  #xrpl-careers {
+    padding-bottom: 5rem;
+    @include media-breakpoint-up(md) {
+      padding-top: 104px;
+      padding-bottom: 104px;
+    }
+  }
+
+
 }
 
 .page-community {
-  @each $platform in "github", "twitch", "stack-overflow", "twitter" {
+  @each $platform in "github", "twitch", "stack-overflow", "twitter", "discord", "youtube", "devto" {
```

Comment:
These logos don't seem to display centered in the circle (on both Firefox and Chromium for desktop at least). The live site also has this problem.

Screenshot of what I'm seeing:

![2022-06-08_131323_056949594](https://user-images.githubusercontent.com/7515597/172710005-b3ea0db6-221d-4670-8db1-b58d0a4ab528.png)

The same CSS (`circled-logo` class) is used on the Code Samples page and the Docs landing page. And the logos look centered on the Code Samples page, so I'm guessing a change we made for the Code Samples page broke it on the Community and Docs pages.


Anyway, we should fix it while we're here.

Reply:
P.S. the Discord logo has very poor contrast on the dark theme. For consistency with the others, the "default" file should be a dark theme version of the logo and we should add a separate file for the light-theme version. See `assets/img/logos/lightmode/stackoverflow.svg` as an example.